ui/src/pages/job/components/JobRunPanel.js

```diff
@@ -0,0 +1,76 @@
+import { DateFromNow, HorizontalDescriptionList } from "@caraml-dev/ui-lib";
+import { EuiHealth, EuiText } from "@elastic/eui";
+import React from "react";
+import { Link } from "react-router-dom";
+import { ConfigSection, ConfigSectionPanel } from "../../../components/section";
+import { JobActions } from "./JobActions";
+
+const JobStatus = ({ status, size }) => {
+  const healthColor = (status) => {
```

Comment:
can this be using `ui/src/pages/job/components/JobStatus.js` or refactored together?